packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -11,6 +11,24 @@ const OVSDB_PORT = 6640
 // =============================================================================
 
 export class OvsdbClient {
+  /*
+  Create an SSL connection to an XCP-ng host.
+  Interact with the host's OpenVSwitch (OVS) daemon to create and manage the virtual bridges
+  corresponding to the private networks with OVSDB (OpenVSwitch DataBase) Protocol.
+  See:
+  - OVSDB Protocol: https://tools.ietf.org/html/rfc7047
+  - OVS Tunneling : http://docs.openvswitch.org/en/latest/howto/tunneling/
+
+  Attributes on created OVS bridges and ports
```

Comment:
What are `bridges` and `ports` in XAPI?

Reply:
A `bridge` would correspond to `network` in XAPI, a `port` to a `PIF` of a `VIF`.

Reply:
Then please, make this clear.

Reply:
Added for `port` and removed reference of `bridge`.
`interface` is internal to openvswitch and doesn't correspond to anything in XAPI.